### DISCUSSION TITLE
zoteor 7.0.0-beta.55+9151fb355 (64-bit) style插件与条目冲突，求助！

### DISCUSSION THREAD

**Discussion by NeilHzz** [MuiseDestiny/zotero-style#843] [Q&A]
![微信截图_20240129150408](https://github.com/MuiseDestiny/zotero-style/assets/116190702/6fe9fa1b-bfcc-41ab-83f2-d81e87491c80)
当插件禁用后条目前面的下拉按钮才会显示
![微信截图_20240129150346](https://github.com/MuiseDestiny/zotero-style/assets/116190702/c808b3c3-193d-42be-9ca6-b64706db032e)
当插件开启后按钮消失，请问怎么解决呢？

**Comment by yunmiumiu** [comment]
换一个项目到第一列试一下，或者可以找beta.54版，先不更新等适配
